### PR TITLE
add support for setting --tls-verify=false for container registry ope…

### PR DESCRIPTION
**Pull request by k-rister**
…rations

- this allows for insecure registries to be used, although there are
  other details that may be required such as endpoint support

### CODE REVIEW

File: tests/test-installer [hunk 1]
```diff
@@ -22,6 +22,13 @@ sudo ./crucible-install.sh \
   --client-server-auth-file /tmp/auth-file.json
 test "$?" = "$ec" || exit 1
 
+# invalid tls verify value
+ec=$(grep EC_TLS_VERIFY_ERROR= crucible-install.sh | cut -d '=' -f2)
+sudo ./crucible-install.sh \
+  --client-server-registry myregistry.io/crucible \
+  --client-server-tls-verify foo
```
Comment (rafaelfolco) [inline tests/test-installer:29]: negative test, that's good :)

File: crucible-install.sh [hunk 2]
```diff
@@ -23,14 +23,19 @@ EC_FAIL_INSTALL=5
...
 EC_FAIL_REGISTRY_SET=12
 EC_FAIL_AUTH_SET=13
 EC_FAIL_CHECKOUT=14
 EC_PUSHD_FAIL=15
 
+# remove a previous installation log
+if [ -e ${GIT_INSTALL_LOG} ]; then
+    rm ${GIT_INSTALL_LOG}
+fi
+
```
Comment (rafaelfolco) [inline crucible-install.sh:38]: why? what does it have to do with tls ?

File: crucible-install.sh [hunk 3]
```diff
@@ -142,6 +147,7 @@ function usage {
 
     optional:
         --client-server-auth-file <authentication file>
+        --client-server-tls-verify true|false
```
Comment (rafaelfolco) [inline crucible-install.sh:150]: --client-server-tls-verify  could be a positional arg

File: tests/test-installer (hunk 1 above)
Comment (rafaelfolco) [reply tests/test-installer:29]: missing a successful case at least

**Review by rafaelfolco — APPROVED** [review]
lgtm, minor things...

File: crucible-install.sh (hunk 2 above)
Comment (k-rister) [reply crucible-install.sh:38]: Nothing.  Except I was initially getting a totally misleading error log because it was discovering an old one :(

**Pull request merged**